dash/test/TeamLocalityTest.cc

```diff
@@ -15,9 +16,15 @@ void print_locality_domain(
     return;
   }
 
-  std::string context_pref = "TeamLocalityTest.print_domain.";
+  return;
```

Comment:
Ah, that's semi-intentional. The function is not needed any longer and will be removed.